http-transport-binding.md

```diff
@@ -200,28 +199,29 @@ Examples:
 Note: per the [HTTP](https://tools.ietf.org/html/rfc7230#section-3.2)
 specification, header names are case-insensitive.
 
-##### 3.1.3.2 HTTP Header Values
+##### 3.1.3.2. HTTP Header Values
 
 The value for each HTTP header is constructed from the respective attribute
 type's [canonical string representation][ce-types].
 
 Some CloudEvents metadata attributes can contain arbitrary UTF-8 string content,
-and per [RFC7230 Section 3][rfc7230-section-3], HTTP headers MUST only use
+and per [RFC7230, section 3][rfc7230-section-3], HTTP headers MUST only use
 printable characters from the US-ASCII character set, and are terminated by a
-CRLF sequence.
+CRLF sequence with optional whitespace around the header value.
 
-Therefore, and analog to the encoding rules for Universal character set host
-names in URIs [RFC3986 3.2.2][rfc3986], the string value MUST be further encoded
-as follows:
+String values which contain Unicode characters outside the ASCII range MUST be
```

Comment:
we agreed on the call yesterday to remove the `which contain Unicode characters outside the ASCII range` text.